text/0033-view-hierarch.md

```diff
@@ -0,0 +1,41 @@
+* Start Date: 2022-11-02
+* RFC Type: feature
+* RFC PR: [#33](https://github.com/getsentry/rfcs/pull/33)
+
+# Summary
+
+Allow SDKs to send View hierarch of the application UI.
+
+# Motivation
+
+Being able to see what the user was looking at is a good way to found out problems in the application. 
+This RFC proposes a way to send information from the UI state of an application, known as "view hierarch".
+
+# Proposal 
+
+The proposal is to create a new envelope item in the form of a json representation of the UI elements, like so: 
```

Comment:
I'd write down the average of such view hierarchy size for a simple screen just for having an idea, Ingest team would really appreciate this info.

Reply:
I dont have data to back this up. View hierarchy can vary a lot.

Reply:
That's why a sample with a simple screen already helps, right now I don't know if it's 1KB/1MB/10MB/100MB.